File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/batch_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class BatchResponse:
    """批量任务响应"""
    id: str = None
    object: str = None
    endpoint: str = None
    errors: Errors = None
    status: str = None
    metadata: Dict[str, str] = None
    input_file_id: str = None
    output_file_id: str = None
    error_file_id: str = None
    created_at: int = None
    completion_window: str = None
    expires_at: int = None
    started_at: int = None
    completed_at: int = None
    failed_at: int = None
    expired_at: int = None
    cancelled_at: int = None
    request_counts: RequestCounts = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/BatchResponse.kt
package com.sdkwork.ai

data class BatchResponse(
    val id: String? = null,
    val object: String? = null,
    val endpoint: String? = null,
    val errors: Errors? = null,
    val status: String? = null,
    val metadata: Map<String, String>? = null,
    val inputFileId: String? = null,
    val outputFileId: String? = null,
    val errorFileId: String? = null,
    val createdAt: Int? = null,
    val completionWindow: String? = null,
    val expiresAt: Int? = null,
    val startedAt: Int? = null,
    val completedAt: Int? = null,
    val failedAt: Int? = null,
    val expiredAt: Int? = null,
    val cancelledAt: Int? = null,
    val requestCounts: RequestCounts? = null
)
